fil_fungible_token/src/token/mod.rs

```diff
@@ -101,20 +102,38 @@ where
         self.state = mutable_state;
         Ok(res)
     }
+
+    /// Resolves an address to an ID address, sending a message to initialise an account there if
+    /// it doesn't exist
+    ///
+    /// If the account cannot be created, this function returns an error
+    fn resolve_to_id_addr(&self, initial_holder: &Address) -> Result<ActorID> {
+        let holder_id = match self.msg.resolve_id(initial_holder) {
+            Ok(addr) => addr,
+            Err(_) => self.msg.initialise_account(initial_holder)?,
+        };
+        Ok(holder_id)
+    }
+
+    /// Attempts to resolve an address to an ID address, returning an error if it did not exist
+    fn get_id_addr(&self, initial_holder: &Address) -> Result<ActorID> {
```

Comment:
Nit: perhaps drop `addr` from the end of these names since they return IDs.
- resolve_addr_id
- get_addr_id
?